scripts/etcd_version_annotations.txt

```diff
@@ -166,6 +166,7 @@ etcdserverpb.HashKVRequest.revision: ""
 etcdserverpb.HashKVResponse: "3.3"
 etcdserverpb.HashKVResponse.compact_revision: ""
 etcdserverpb.HashKVResponse.hash: ""
+etcdserverpb.HashKVResponse.hash_revision: ""
```

Comment:
I think that this field should be annotated that it was added in v3.6. 
Seems it requires a proto.option in rpc.proto file.

Reply:
Thanks @ptabor for pointing out this. 

But this field isn't persisted in the WAL, nor the boltDB. Should we still annotate that it's added in v3.6? @serathius @ptabor

Reply:
Added annotation `[(versionpb.etcd_version_field)="3.6"]` anyway although it isn't necessary for now. But it will not do any harm for now, and it is also safe in case it will be persisted in WAL in future.